src/client/sound.h

```diff
@@ -19,12 +19,14 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #pragma once
 
-#include "irr_v3d.h"
+#include "irrlichttypes_extrabloated.h"
 #include <limits>
 #include <string>
 #include <unordered_map>
 #include <unordered_set>
 #include <vector>
+#include "settings.h"
+#include "util/numeric.h"
 
```

Comment:
Now that the implementation of `sound_control_by_window` has been moved to the .cpp file, the two includes above are no longer necessary in the header file.

Reply:
Those are used in `src/client/sound.cpp`. I will move those includes there.